packages/web/modals/add-liquidity.tsx

```diff
@@ -50,12 +51,13 @@ export const AddLiquidityModal: FunctionComponent<
 
   return (
     <ModalBase
-      title={t("addLiquidity.title")}
+      // title={t("addLiquidity.title")}
       {...props}
       isOpen={props.isOpen && showModalBase}
+      hideCloseButton
+      className="!max-w-[57.8125rem]"
```

Comment:
Obviously, since we're given the pool ID, we'll have to eventually query the pool for whether it's a CL pool or balancer/stableswap then show the right view.